File: Programming Basics/C++/For loops/CleverLily/CleverLily.cpp
#include <iostream>
using namespace std;

int main()
{
    int age, toyPrice;
    double washingMachinePrice;
    
    cin >> age >> washingMachinePrice >> toyPrice;

    int savedMoney = 0, moneyToReceive = 10;
    for (int i = 1; i <= age; i++)
    {
        if (i % 2 == 0)
        {
            savedMoney += moneyToReceive - 1;
            moneyToReceive += 10;
        }
        else { savedMoney += toyPrice; }
    }

    cout.setf(ios::fixed);
    cout.precision(2);

    if (savedMoney >= washingMachinePrice) cout << "Yes! " << savedMoney - washingMachinePrice << endl;
    else cout << "No! " << washingMachinePrice - savedMoney << endl;
}

File: Programming Basics/JavaScript/For loops/Exercises/clever-lily.js
function solve(input) {
    const age = Number(input[0]);
    const washingMachinePrice = Number(input[1]);
    const toyPrice = Number(input[2]);

    let savedMoney = 0;
    let moneyToReceive = 10;
    for (let i = 1; i <= age; i++) {
        if (i % 2 == 0) {
            savedMoney += moneyToReceive - 1;
            moneyToReceive += 10;
        } else {
            savedMoney += toyPrice;
        }
    }

    if (savedMoney >= washingMachinePrice) console.log(`Yes! ${(savedMoney - washingMachinePrice).toFixed(2)}`);
    else console.log(`No! ${(washingMachinePrice - savedMoney).toFixed(2)}`);
}
